Sources/Core/Public/GTMSessionFetcher/GTMSessionFetcher.h

```diff
@@ -1103,6 +1104,10 @@ __deprecated_msg("implement GTMSessionFetcherAuthorizer instead")
 // will not be called.
 - (void)stopFetching;
 
+// Cancel the fetch of the request that's currently in progress.  The completion handler
+// is called on option.
+- (void)stopFetching:(BOOL)shouldCallCallbacks;
```

Comment:
My understanding is that this change is being driven by desire for Swift `async` support, and if Swift code demands the continuation is always called then the API should make sure than an appropriately Swift-configured fetcher will always trigger callbacks on `stopFetching` being called, without the code canceling the fetch needing to know to call a different `stopFetching` variant.

I think this needs to be reconsidered and made into a single fire-and-forget kind of approach, where Swift code can set a `fetcher.stopFetchingTriggersCallbacks = true` kind of property, at which point any call to `stopFetching` will trigger the callbacks with the appropriate error.